compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1946,7 +1946,7 @@ error.unsupported.exposure.of.construct.from.module.generated.for.client.decl=\
   exposing a construct from a module generated for a client declaration is not yet supported
 
 error.module.generated.for.client.decl.must.have.a.client.object.type=\
-  a module generated for a client declaration must have an object type or class named ''client''
+  a module generated for a client declaration must have an object type or class named ''Client''
```

Comment:
You will also have to change this to get the tests passing: https://github.com/ballerina-platform/ballerina-lang/commit/9757d0847d626bb3d98d6e25a062e4875c13d0de

Reply:
Updated.